compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+  web:
+    image: datafed-ws
+    depends_on:
+      - arangodb
+    ports:
+      - '80:80'
+      - '443:443'
```

Comment:
The host ports will need to be configurable. Just keep that in mind. We can create an .env file with defaults in the same folder as the compose file to do this. Also, I recommend putting compose.yml in a compose folder instead of directly in the root of the project to keep things clean.